src/components/Theme/ScreenLayout.tsx

```diff
@@ -0,0 +1,18 @@
+import { h, Fragment, ComponentType, ComponentChildren, VNode } from 'preact'
+import theme from './style.scss'
+
+type ScreenLayoutProps = {
+  children: ComponentChildren
+  actions?: ComponentType | VNode
```

Comment:
`ComponentType` is a generic type to infer `type of a component`, while `VNode` is a renderable node. So if you want to directly render `actions` inside this component, `VNode` is the correct type, while with `ComponentType`, you need to render it as `<actions />`.

```suggestion
  actions?: VNode
```